Fix nav-bar Share overlapping other dropdowns

Noviv opened the pull request:
Turns out `click.editable` isn't fired when another dropdown is clicked, meaning this callback is never called. When changed to `mouseup`, the callback is called and the Share menu is successfully closed.

Also, it looks like this could also be solved by specifiying the layout properly in `views/index.pug` (like the Other and Policies dropdowns are specified), but that seemed like a larger fix. Plus this got the job done :)

Fixes #1011

Noviv commented:
This should fix #1011 but I wasn't sure how to reference it properly, apologies!

RabsRincon commented:
Hi @Noviv thanks for the fix!

For the record, GitHub recognizes some keywords to close issues, you can read more at https://help.github.com/articles/closing-issues-using-keywords/. I've updated your pr text with one so you can see it in action.

Did you check that it does not have any unwanted effects? I'd expect this callback to be run more often than the previous one, for more elements.

Noviv commented:
@RabsRincon I couldn't find any unexpected behaviour when I changed it to `mousedown`, and I'm not sure it'll be called any more than `click` seeing as `mousedown`/`mouseup`/`click` must be called together. It might be better to switch to `mousedown` as when the user clicks somewhere else the Share dropdown should disappear immediately, not when they release - your call!

I also pushed one more change: the Other and Policies popovers seem to toggle, whereas Share was hiding + re-rendering on each click, I just added a simple toggle to fix this :)

RabsRincon commented on static/sharing.js:
```diff
@@ -124,7 +124,13 @@ function initShareButton(getLink, layout) {
         trigger: 'manual'
     }).click(function () {
-        getLink.popover('show');
+        // load opposite state on click to prevent unnecessary popover rendering
```
Shouldn't `.popover('toggle')` do the same, but with less boilerplate?

Noviv replied:
Totally right, I should've done a little more research haha - commit coming right up!

RabsRincon commented:
So, it works like a charm, which is great!

Thanks for the fix. It should hit the live site "soon". I'll be sure to let you know when it does :)
